torch/csrc/jit/passes/quantization.cpp

```diff
@@ -630,6 +630,29 @@ graph(%self, %a, %b, %inplace):
   };
 };
 
+std::vector<Node*> getCallMethods(Value* v) {
```

Comment:
this is not getting call method for value `v`, but getting all call method in the graph.

to get call method for v, you just need to look at uses of v and check if the node is a `CallMethod` or not